src/frontend/src/lib/components/settings/EnabledNetworksPreviewIcons.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+	import { SUPPORTED_NETWORKS } from '$env/networks/networks.env';
+	import NetworkLogo from '$lib/components/networks/NetworkLogo.svelte';
+	import { logoSizes } from '$lib/constants/components.constants';
+	import { userNetworks } from '$lib/derived/user-networks.derived';
+	import type { Network } from '$lib/types/network';
+	import type { UserNetworks } from '$lib/types/user-networks';
+
+	const numberOfIcons = 4;
+
+	const getEnabledList = (networks: UserNetworks) =>
```

Comment:
let's add a return type
```suggestion
	const getEnabledList = (networks: UserNetworks): Network[] =>
```